perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -303,62 +336,72 @@ int main(int ArgCount, char** Args)
     instruction_table Table;
     Sim86_Get8086InstructionTable(&Table);
     printf("8086 Instruction Instruction Encoding Count: %u\n", Table.EncodingCount);
-  
-    if (ArgCount > 1)
-    { 
-        for (int ArgIndex = 1; ArgIndex < ArgCount; ArgIndex++)
-        {
-            s16 Registers[RegisterCount] = {};
-            bool FlagArray[Flag_count] = {};
 
-            std::string OutputBuffer;
-            char* FileName = Args[ArgIndex];
-            std::ifstream File;
-            File.open(FileName, std::ifstream::binary | std::ifstream::in);
-            if (!File.good())
-            {
-                std::cout << "Error opening file " << FileName;
-                return -1;
-            }
-            
-            std::cout << "\n" << FileName << "\n";
-            std::vector<u8> Bytes((std::istreambuf_iterator<char>(File)), std::istreambuf_iterator<char>());
-            const u16 BytesRead = static_cast<u16>(Bytes.size());
-            u16& InstructionPointer = reinterpret_cast<u16&>(Registers[IPRegister]);
+    Memory = reinterpret_cast<u8*>(malloc(Megabyte));
```

Comment:
Funnily enough, I elected to use malloc because that's what Casey did even though I wanted to use a fixed size array.
Fixed
3446096e8be08fed3c8096a1c73251505b2c1509